plugins/git/README.md

```diff
@@ -10,202 +10,202 @@ plugins=(... git)
-| gwch                 | git whatchanged -p --abbrev-commit --pretty=medium                                                                                                                                       |
-| gwip                 | git add -A; git rm $(git ls-files --deleted) 2> /dev/null; git commit --no-verify --no-gpg-sign --message "--wip-- [skip ci]"                                                            |
-| gam                  | git am                                                                                                                                                                                   |
-| gamc                 | git am --continue                                                                                                                                                                        |
-| gams                 | git am --skip                                                                                                                                                                            |
-| gama                 | git am --abort                                                                                                                                                                           |
-| gamscp               | git am --show-current-patch                                                                                                                                                              |
-| gwt                  | git worktree                                                                                                                                                                             |
-| gwtls                | git worktree list                                                                                                                                                                        |
-| gwtmv                | git worktree move                                                                                                                                                                        |
-| gwtrm                | git worktree remove                                                                                                                                                                      |
+| Alias                | Command                                                                                                                                                         |
+| :------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| grt                  | cd "$(git rev-parse --show-toplevel &#124;&#124; echo .)"                                                                                                       |
```

Comment:
Could we move this ones manually to its proper place?

Reply:
You mean this should be right after

```
| gunwip | git rev-list --max-count=1 --format="%s" HEAD &#124; grep -q "--wip--" && git reset HEAD~1 |
```
?